omise/omise-android
# Sample app in Kotlin

### Objective

To add a sample app for using SDK to users that implement they's apps in Kotlin.

### Description of changes

In this PR add build variants config and create package directories for Java and Kotlin to able to choose source code between Java and Kotlin. Also, created the `CheckoutActivity` in Kotlin version. 

### QA

To test this, you can choose build variant by click **Build Variants** it will show the build variant options. So, you can choose **javaDebug** to use the sample app from Java code or choose **kotlinDebug** for Kotlin code. 

There is no difference in the interface, and it should work in the same indifference version.

![Screen Shot 2562-09-27 at 11 30 20](https://user-images.githubusercontent.com/2638321/65742913-72086f00-e11c-11e9-9e1b-9092d55043e7.png)

![Screen Shot 2562-09-27 at 11 30 34](https://user-images.githubusercontent.com/2638321/65742888-53a27380-e11c-11e9-8610-4add26d6f53c.png)
